assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -84,18 +85,25 @@ function MyStories() {
   const [status, setStatus] = useState(STORY_STATUSES[0].value);
   const [typeaheadValue, setTypeaheadValue] = useState('');
   const [viewStyle, setViewStyle] = useState(VIEW_STYLE.GRID);
+  const [currentPageToFetch, setCurrentPageToFetch] = useState(1);
+  const [allPagesLoaded, setAllPagesLoaded] = useState(false);
   const [currentStorySort, setCurrentStorySort] = useState(
     STORY_SORT_OPTIONS.LAST_MODIFIED
   );
   const [currentListSortDirection, setListSortDirection] = useState(
     SORT_DIRECTION.ASC
   );
+
+  const [debounceTypeaheadValue] = useDebouncedCallback((newTypeaheadValue) => {
+    setTypeaheadValue(newTypeaheadValue);
+  }, 800);
```

Comment:
can probably just make this shorter if you like it better. If you think it's urgly feel free to stick with current implementation
```
const [debounceTypeaheadValue] = useDebouncedCallback(setTypeaheadValue, 800);
```